addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -969,6 +969,12 @@ const Wysiwyg = Widget.extend({
             const {oeModel: resModel, oeId: resId} = editableEl.dataset;
             const proms = [...editableEl.querySelectorAll('.o_modified_image_to_save')].map(async el => {
                 const isBackground = !el.matches('img');
+                if (!el.closest(".o_dirty").isEqualNode(editableEl)) {
```

Comment:
Works fine :+1: 
But unless there's a reason for it I didn't see, I think a strict equality would be better suited than `isEqualNode()`. :thinking:

Reply:
Indeed, you are right :+1: